apps/antalmanac/src/components/dialogs/CopySchedule.tsx

```diff
@@ -70,6 +74,7 @@ function CopyScheduleDialog(props: CopyScheduleDialogProps) {
                                 {name}
                             </MenuItem>
                         ))}
+                        <MenuItem value={scheduleNames.length + 1}>Copy to New Schedule</MenuItem>
```

Comment:
```suggestion
                        <MenuItem value={'copyToNew'}>Copy to New Schedule</MenuItem>
```
issue: I think it's confusing for us to declare an item with a value that is greater than the length of the elements. 

suggestion: set the value to something more explicit. Then, you can extract the double if statements upwards and minimize nesting.